Gem/Code/Source/AutoGen/GemSpawnerComponent.AutoComponent.xml

```diff
@@ -24,4 +24,20 @@
 
     <ArchetypeProperty Type="AZStd::string" Name="GemSpawnTag"
                        ExposeToEditor="true" Description="A tag value for entities the describe the location of gems spawn points." />
+
+    <RemoteProcedure Name="RPC_SpawnGem" InvokeFrom="Server" HandleOn="Authority" 
+                     IsPublic="true" IsReliable="true" GenerateEventBindings="false" Description="Spawn a gem at a specific location">
```

Comment:
Maybe set the generate event bindings to true so users can spawn gems from script too?